### PR TITLE
Added Rating component.

### PR DESCRIPTION
**Pull request by sameep-s**
Added rating component to the library.

click [here](https://deploy-preview-14--aphro-css.netlify.app/components/rating.html) to see the demo.

### CODE REVIEW

**Review by harshparab — APPROVED** [review]
Looks good to merge

File: css/comp.css [hunk 1]
```diff
@@ -1,6 +1,7 @@
 @import "./core.css";
 @import "./docPage.css";
 @import "./util.css";
+@import "./rating.css";
 
 .exapmle-container {
   background-color: white;
```
Comment (arjunbharti) [inline css/comp.css:7]: you can use color variable here

File: css/rating.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
+.rating-wrapper {
+  width: 100%;
+  display: flex;
+  transition: 360ms;
+}
+
+.rating-wrapper > div {
+  font-size: 3rem;
+  color: #808080;
```
Comment (arjunbharti) [inline css/rating.css:9]: use color variable here

File: css/rating.css [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+}
+
+.rating-wrapper > div:hover {
+  color: yellow;
```
Comment (arjunbharti) [inline css/rating.css:13]: color varaible here too

**Review by arjunbharti — APPROVED** [review]
Try to use color variables in the css, everything else is fine!

**Pull request merged**